index.html

```diff
@@ -55,92 +91,162 @@ <h2 class="heed">Multi-Post Stories</h2>
+    </section>
+  
+    <section class="projects">
+      <h2>Professional Art Printing Data</h2>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="sites1">
+        <ul>
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button type="button" class="button2">See Project</button>  
+    </section>
+  
+    <section class="projects">
+      <h2>Professional Art Printing Data</h2>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="sites1">
+        <ul>
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button type="button" class="button2">See Project</button>  
+    </section>
+  
+    <section class="projects">
+      <h2>Professional Art Printing Data</h2>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <div class="sites1">
+        <ul>
+          <li><a href="#">html</a></li>
+          <li><a href="#">bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button type="button" class="button2">See Project</button>  
+    </section>
   </section>
 
-  <section class="projects">
-    <h2>Professional Art Printing Data</h2>
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="sites1">
-      <ul>
-        <li><a href="#">html</a></li>
-        <li><a href="#">bootstrap</a></li>
-        <li><a href="#">Ruby</a></li>
-      </ul>
+  <section class="projects_desktop">
+    <div class="project1">
+        <button type="button" class="button_project">See Project</button>  
     </div>
-    <button type="button" class="button2">See Project</button>  
-  </section>
-
-  <section class="projects">
-    <h2>Professional Art Printing Data</h2>
-    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-    <div class="sites1">
-      <ul>
-        <li><a href="#">html</a></li>
-        <li><a href="#">bootstrap</a></li>
-        <li><a href="#">Ruby</a></li>
-      </ul>
+    <div class="project2">
+        <h2>Data Dashboard Healthcare</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="sites1">
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] For this part, not all the projects are showing. Kindly fix the issue.
![image](https://user-images.githubusercontent.com/43698511/130281383-37af904b-258f-41dd-9e72-d2e65ba91de6.png)

- [ ] In the original design. All the featured projects have the description on top of their image with a dark opacity with white color text. When the project is hovered on with the mouse the description and dark opacity disappear and a See Project button appears. Kindly fix the issue.
![image](https://user-images.githubusercontent.com/43698511/130281979-44812061-cd82-4f43-8961-99df40e5a6dd.png)